internal/tools/semconv-gen/generator.go

```diff
@@ -67,17 +73,28 @@ type config struct {
 	outputFilename   string
 	templateFilename string
 	containerImage   string
+	specVersion      string
```

Comment:
Should a flag for the spec version be added as well, doing the lookup logic if it's not provided?  I really hope we never have to re-generate prior versions, but it could be a simple change now to provide the option.

Reply:
Done. PTAL.